src/components/create-filter-form.js

```diff
@@ -86,12 +89,13 @@ const CreateFilterDialog = ({
     title,
     customTextValidationBtn,
     customTextCancelBtn,
-    action,
+    activeDirectory,
```

Comment:
this activeDirectory is in the redux/store, you could avoid passing it in props
`const activeDirectory = useSelector((state) => state.activeDirectory);`

Reply:
done